logic/atrium-logic-jvm/src/main/kotlin/ch/tutteli/atrium/logic/impl/DefaultChronoLocalDateAssertions.kt

```diff
@@ -47,4 +48,38 @@ class DefaultChronoLocalDateAssertions : ChronoLocalDateAssertions {
         expected: ChronoLocalDate
     ): Assertion = container.createDescriptiveAssertion(SAME_DAY, expected) { it.isEqual(expected) }
 
+    override fun <T : ChronoLocalDate> isBefore(
+        container: AssertionContainer<T>,
+        expected: String
+    ): Assertion = container.createDescriptiveAssertion(IS_BEFORE, expected) {
+        it.isBefore(LocalDate.parse(expected))
+    }
```

Comment:
please delegate to the method which expects a ChronoLocalDate instead of duplicating the logic
```suggestion
    ): Assertion = container.isBefore(LocalDate.parse(expected))
```